astroquery/esa_hubble/tests/test_esa_hubble.py

```diff
@@ -0,0 +1,127 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""
+
+@author: Javier Duran
+@contact: javier.duran@sciops.esa.int
+
+European Space Astronomy Centre (ESAC)
+European Space Agency (ESA)
+
+Created on 13 Ago. 2018
+
+
+"""
+import unittest
+
+from astroquery.esa_hubble.core import ESAHubbleClass
+from astroquery.esa_hubble.tests.dummy_handler import DummyHandler
+from astroquery.esa_hubble.tests.dummy_tap_handler import DummyESAHubbleTapHandler
+from astropy import coordinates
+
+
+class TestESAHubble(unittest.TestCase):
+
+    def test_get_product(self):
+        parameterst = {}
+        parameterst['query'] = "select top 10 * from hsc_v2.hubble_sc2"
+        parameterst['output_file'] = "test2.vot"
+        parameterst['output_format'] = "votable"
+        parameterst['verbose'] = False
+        dummyTapHandler = DummyESAHubbleTapHandler("launch_job", parameterst)
+
+        parameters = {}
+        parameters['observation_id'] = "J6FL25S4Q"
+        parameters['calibration_level'] = "RAW"
+        parameters['verbose'] = False
+        dummyHandler = DummyHandler("get_product", parameters)
+        ehst = ESAHubbleClass(dummyHandler, dummyTapHandler)
+        ehst.get_product("J6FL25S4Q", "RAW")
```

Comment:
This test, and the other tests in this file, look like they require an internet connection to work.  You need to mark these with the `@remote_data` decorator.  See https://astroquery.readthedocs.io/en/latest/testing.html#test-module-remote-py